### PR TITLE
Calculations

### PR DESCRIPTION
**Pull request by Glassinatorn**
Hopefully finished with function for estimating the amount of water per household.

Going to create new branch for next feature.

### CODE REVIEW

File: flowClalc.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<html>
+	<body>
+		<script src="flowClalc.js"></script>
+		<script>
+			var houses = [
```
Comment (SpaceLenore) [inline flowClalc.html]: - indentation seems to be more than 4 spaces
- use let or const instead of var

File: flowClalc.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+						address: "skolvägen 5",
+						nrOfInd: 4,
+						Qd: 300
+				}];
+
+			//var dia = 0.2;
+			//var friction = 0.01;
+			//var fall = 0.01;
+			//var Q = darcy(dia, friction, fall);
+			var result = est_water(houses);
```
Comment (SpaceLenore) [inline flowClalc.html]: * use let or const instead of var

File: flowClalc.js [hunk 3]
```diff
@@ -1,16 +1,44 @@
...
+ * houses.
+ * houses = array of houses to calc on, Qd = waste water per day, p = number of
+ * people in the house, Cd = max factor per day, Cd = max factor per hour
  * */
-function houseWater (house, Qd = 300, p = 100, Cd = 1.6, Ct = 1.7, QInd = 0) {
-	var Qs = ((Qd * p)/(3600 * 24)) * Cd * Ct + QInd;
-	return Qs;
-}
+const est_water = (houses, Cd = 1, Ct = 1, QInd = 0) => {
+		var Qs = 0;
```
Comment (SpaceLenore) [inline flowClalc.js]: * use let or const instead of var

File: flowClalc.js [hunk 4]
```diff
@@ -1,16 +1,44 @@
...
+		return Qs;
+};
+
+/* To calc how much speed there will be in the pipe depending on the diameter of
+ * the pipe, the friction of the pipe, as well as the fall, which should not be
+ * more than 6-12 promille.
+ * This function gives the speed in meters/s.
+ * */
+const darcyflow3 = (dia, friction = 0.01, fall = 0) => {
+		var Q = (((3.14159 * (dia * dia))/4) * Math.sqrt((2 * 9.82 * dia * fall)/friction))
```
Comment (SpaceLenore) [inline flowClalc.js]: * use let or const instead of var

File: flowClalc.js [hunk 5]
```diff
@@ -1,16 +1,44 @@
...
-function darcy (dia, friction = 0.01, fall = 0) {
-	console.log(dia);
-	var Q = (((3.14159 * (dia * dia))/4) * Math.sqrt((2 * 9.82 * dia * fall)/friction))
-	return Q;
-}
+ * more than 6-12 promille.
+ * This function gives the speed in cubic meters/s.
+ * */
+const darcyflow = (dia, friction = 0.01, fall = 0) => {
+		var Q = Math.sqrt((2 * 9.82 * dia * fall)/friction)
```
Comment (SpaceLenore) [inline flowClalc.js]: * use let or const instead of var

File: flowClalc.js [hunk 6]
```diff
@@ -1,16 +1,44 @@
...
+};
+
+/* Function to calc the resistance coefficient */
+const coeff = (friction = 0.01) => {
+		return /* calc */
+};
+
+/* To calc the loss of energy when pumping up water verticaly */
+const darcyheight = (dia, friction = 0.01, length = 5, speed = 1) => {
+		var hf = ((friction * length)/dia + ceoff(friction)) * ((speed * speed)/(2 * 9.82))
```
Comment (SpaceLenore) [inline flowClalc.js]: * use let or const instead of var

File: flowClalc.html (hunk 1 above)
Comment (Glassinatorn) [reply flowClalc.html]: have changed array to const and other variables to let

File: flowClalc.js (hunk 3 above)
Comment (Glassinatorn) [reply flowClalc.js]: done

File: flowClalc.js (hunk 4 above)
Comment (Glassinatorn) [reply flowClalc.js]: done

File: flowClalc.js (hunk 6 above)
Comment (Glassinatorn) [reply flowClalc.js]: done

File: flowClalc.js (hunk 5 above)
Comment (Glassinatorn) [reply flowClalc.js]: done

**Pull request merged**